test/e2e/util.go

```diff
@@ -1137,6 +1157,20 @@ func RunRC(config RCConfig) error {
 
 		Logf("%v %v Pods: %d out of %d created, %d running, %d pending, %d waiting, %d inactive, %d unknown ",
 			time.Now(), rc.Name, len(pods), config.Replicas, running, pending, waiting, inactive, unknown)
+
+		runningMetric.Set(float64(running))
+		pendingMetric.Set(float64(pending))
+
+		if testContext.PrometheusPushGateway != "" {
```

Comment:
I'll look into a more modular implementation and feel free to punt to post 1.0 I understand the churn issue